openslides_backend/action/actions/group/delete.py

```diff
@@ -14,3 +22,159 @@ class GroupDeleteAction(DeleteAction):
     model = Group()
     schema = DefaultSchema(Group()).get_delete_schema()
     permission = Permissions.User.CAN_MANAGE
+
+    def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
+        instance = super().update_instance(instance)
+        group = self.datastore.get(
+            FullQualifiedId(Collection("group"), instance["id"]), []
+        )
+        self.mediafile_ids: List[int] = list(
+            (set(group.get("mediafile_access_group_ids", set())) or set())
+            | (set(group.get("mediafile_inherited_access_group_ids", set())) or set())
```

Comment:
If I counted the brackets correctly, you write here `set(...) or set()`, so you should be able to remove the second `set()`

Reply:
done